### ISSUE TITLE
About the result of lora fine-tuning

### ISSUE THREAD

**Issue by lucasliunju** [git-cloner/llama-lora-fine-tuning#15]
May I ask the result of this lora fine-tuning on MMLU task.

Thanks! 

Best,
Lucas

**Comment by bing0037** [comment]
Same question. Could you provide the **best result** with LLaMA 1 or LLaMA2 on Vicuna and give the **hyperparameters** (num of epoch/learning rate/lora_r/batch_size)? 
I wanna reproduce your result ans use it as the baseline. Training loss or MMLU or alpaca_eval results are appreciated!

**Comment by bing0037** [comment]
@little51 Could you please provide any information?
